sycl/unittests/helpers/PiImage.hpp

```diff
@@ -160,7 +160,39 @@ template <typename T> class PiArray {
 /// Convenience wrapper for pi_device_binary_property_set.
 class PiPropertySet {
 public:
-  PiPropertySet() = default;
+  PiPropertySet() {
+    // Most of unit-tests are statically linked with SYCL RT. On Linux and Mac
+    // systems that causes incorrect RT installation directory detection, which
+    // prevents proper loading of fallback libraries. See intel/llvm#6945
```

Comment:
I've seen an idea (probably in the internal bug tracker) to make those spv part of .so binary image to address this issue too. JFYI.